test/integration/config_4/example_spec.rb

```diff
@@ -0,0 +1,104 @@
+describe package('haproxy') do
```

Comment:
Do we need example4 spec?

Reply:
Nothing else has a `listen` setup to test out the options property. What are we thinking for removing it? Just to get the number of tests down?

Reply:
Yeah, happy to move it into one of the other suites so it gets tested.